ddp-workspace/projects/ddp-osteo/src/app/components/gallery/gallery.component.scss

```diff
@@ -21,7 +21,7 @@
     flex-direction: column;
     justify-content: flex-start;
     grid-area: text;
-    padding: 1.25rem;
+    padding: 4rem 2rem 2rem 2rem;
```

Comment:
![image](https://user-images.githubusercontent.com/30126907/71064006-11b72580-217f-11ea-8a3a-69dfadf20453.png)

Reply:
One thing I just noticed looking at carousel in mobile is that the height of the whole component  changes when you swipe through pictures. Perhaps we ought to fix the height?